### ISSUE TITLE
Can graph embedding be applied to supervised learning?

### ISSUE THREAD

**Issue by fangkuoyu** [benedekrozemberczki/karateclub#79]
I have one question about the use case of graph embedding. In the tutorial for graph embedding, we use the fit method to train a model (e.g., FeatherGraph) with the whole dataset (e.g. Reddit 10K), then we use the get_embedding method to get the embeddings for the whole dataset. Can we train the model with the training dataset then use the model to predict the embeddings for the test set? Because there is no predict function in graph embedding, I am wondering if it is doable in the implementation of karateclub?

**Comment by benedekrozemberczki** [comment]
You mean vector inference? That is not supervised - that is just induction. The current implementation is only transductive. Please feel free to star the repo and hit follow.
